docs/cqd_reference/src/asciidoc/_chapters/query_execution.adoc

```diff
@@ -237,3 +237,27 @@ If you enable small scanner on large size scan incorrectly, then you are likely
 | *Deprecated In Release*     | Not applicable.
 |===
 
+<<<
+[[scratch-disks]]
+== SCRATCH_DISKS
+
+[cols="25%h,75%"]
+|===
+| *Description*               |	Allows a CQD specification to override the env settings for STFS locations.
+| *Values*                    | Scratch disk locations for overflow (linux path names).
```

Comment:
@prashanth-vasudev do you know what the separator between the names is? A blank or comma or something else?